core/src/main/resources/digester-rules.xml

```diff
@@ -7,11 +7,11 @@
 
 	<rule pattern="*/include" factory="nl.nn.adapterframework.configuration.digester.ConfigurationDigesterFactory" selfRegisterMethod="include"/>
 
-	<rule pattern="configuration/jmsRealms" factory="nl.nn.adapterframework.configuration.digester.JmsRealmsFactory"/>
```

Comment:
het idee was dat de frankdoc dit (absolute) pad zou gaan herkennen, om zo de dummy setters weg te werken. Ik denk dat het verstandig is de absolute paden te houden.

Reply:
Heb de absolute paden gehouden, maar wel registerMethods toegevoegd, omdat de FrankDoc anders niet weet wat de kinderen zijn.